### ISSUE TITLE
revise Vitessce view config for sn/scRNA-seq datasets

### ISSUE THREAD

**Issue by ngehlenborg** [hubmapconsortium/portal-ui#2242]
With Azimuth annotations available, the cell type annotations should be made more visible.

Suggested layout:

![image (32)](https://user-images.githubusercontent.com/1216518/142284413-eb59d274-a719-4279-9014-4822aef79026.png)

The cell sets for the cell type should be listed first in the list, so that they show up in the top left corner.

**Comment by ilan-gold** [comment]
@ngehlenborg One thing that just occurred to me - lots of clusters are going to get a bit smushed in the violin plot going horizontally.

**Comment by ngehlenborg** [comment]
Yes, but it would be good to show it. Can we make it scrollable?

**Comment by ilan-gold** [comment]
I think my point was more that making it smaller than it currently is would be not great.  The scrollable idea is good.  I'll make an issue for it in Vitessce.

**Comment by ngehlenborg** [comment]
I think there is not enough space (in the current UI setup) to accommodate a view that is wide enough for all cell types no matter how wide we make it.

**Comment by ngehlenborg** [comment]
We will revisit once #2298 has been resolved.
